server/src/main/java/org/opensearch/gateway/remote/RemoteClusterStateService.java

```diff
@@ -1262,8 +803,19 @@ public String getLastKnownUUIDFromRemote(String clusterName) {
         }
     }
 
+    public RemoteClusterStateCleanupManager getCleanupManager() {
+        return remoteClusterStateCleanupManager;
+    }
+
+    private BlobStoreTransferService getBlobStoreTransferService() {
+        if (blobStoreTransferService == null) {
+            blobStoreTransferService = new BlobStoreTransferService(getBlobStore(), threadpool);
+        }
+        return blobStoreTransferService;
+    }
```

Comment:
Ensure this is static singleton

Reply:
Removed this method and initiating the `blobStoreTransferService` in `start()` method